docs/api.rst

```diff
@@ -25,6 +25,47 @@
 API Docs
 ========
 
-invenio_records_rest
------------------------
+Ext
```

Comment:
@hachreak IMHO we don't need `Ext` section as it's directly starts with the extension module docstring.